interview-questions.md

```diff
@@ -8,36 +8,99 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Student model for all students and their information is assigned a personal id for each student. Through migration, an update can be made and column added for foreign key. Foreign key column allow rails magic to make the associations. For example, Cohort_has_many, foreign keys belongs_to Student.    
 
-Researched answer:
+Researched answer: When creating new student or updating a students information, Rails automatically add primary key to each student in database.
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: RESTful routes, PATCH/PUT, allow users modify things. Params are need to ensure users are authorized to make any changes.
 
-Researched answer:
+Researched answer: Params has access to frorms and URL queries. Params define what is required from a user. Prevents any malicious attempts to pass false data. 
```

Comment:
Nice work on your answer here!  RESTful routes that require params is Show, Update, Edit, and Destroy as we need the id of the instance we want to work with.